### PR TITLE
Add SA cleaner

### PR DESCRIPTION
**Pull request by Halamix2**
**Description**

Changes proposed in this pull request:

- Add cleaner for service-accounts secrets in Secret Manager
- ...
- ...

**Related issue(s)**

- #5748

### CODE REVIEW

**Review by Sawthis — CHANGES_REQUESTED** [review]

File: development/gcp/pkg/secretmanager/service.go [hunk 1]
```diff
@@ -92,3 +122,33 @@ func (sm *Service) GetSecretVersionData(secretPath string) (string, error) {
...
+func (sm *Service) GetAllSecrets(projectPath string, filter string) ([]*gcpsecretmanager.Secret, error) {
+	var secrets []*gcpsecretmanager.Secret
+	nextPageToken := ""
+
+	for {
+		secretListCall := sm.Service.Projects.Secrets.List(projectPath)
+		if filter != "" {
+			secretListCall = secretListCall.Filter(filter)
+		}
+		// TODO support paging
```
[the remark this answers is not in the document]
Comment (Halamix2) [reply development/gcp/pkg/secretmanager/service.go]: Done

**Review by Sawthis — APPROVED** [review]
Approved

**Pull request merged**